Template.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30803.129
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Template", "Template.csproj", "{8A3CECC5-7801-4AB6-B549-F4AE8EFA1D59}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{8A3CECC5-7801-4AB6-B549-F4AE8EFA1D59}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{8A3CECC5-7801-4AB6-B549-F4AE8EFA1D59}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{8A3CECC5-7801-4AB6-B549-F4AE8EFA1D59}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{8A3CECC5-7801-4AB6-B549-F4AE8EFA1D59}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {9A109073-21FC-46C6-A40A-8BE4AD21C24A}
+	EndGlobalSection
+EndGlobal
```

Comment:
Why do we need .sln file at all? I think it should be removed.

Reply:
resolved in https://github.com/GmodNET/Templates/pull/2/commits/b35fbb5c2afa30db85c220f50bde175673437d7c